cli/src/main/java/com/devonfw/tools/ide/io/FileAccessImpl.java

```diff
@@ -659,8 +659,22 @@ public static String generatePermissionString(int permissions) {
   private void extractArchive(Path file, Path targetDir, Function<InputStream, ArchiveInputStream> unpacker) {
 
     this.context.trace("Unpacking archive {} to {}", file, targetDir);
+    // calculate the size of archive for the progressbar max size
+    long sizeOfArchive = 0;
     try (InputStream is = Files.newInputStream(file); ArchiveInputStream ais = unpacker.apply(is)) {
       ArchiveEntry entry = ais.getNextEntry();
+      while (entry != null) {
+        sizeOfArchive += entry.getSize();
+        entry = ais.getNextEntry();
+      }
+      System.out.println("Size of Archive is: " + sizeOfArchive);
```

Comment:
please remove `System.out`. If you want to keep it use logging on debug level.

Reply:
Sorry for that. I used that for testing and forgot to remove. 

done